docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -80,205 +80,93 @@ This is the list of options that can be applied to `exporters.instances` section
 - **Required:** yes
 - **Type:** string
 
-    Set the exporter name.
+: Set the exporter name.
 
 ###### exporter_type
 
 - **Required:** yes
 - **Type:** string
 
-    Set the exporter type. One of `deploytime`, `committime`, `failure`.
+: Set the exporter type. One of `deploytime`, `committime`, `failure`.
 
 ###### env_from_secrets
 
 - **Required:** no
 - **Type:** list
 
-    **Recommended for sensitive data**
-
-    List of secrets, like in the following example.
-    ```yaml
-    env_from_secrets:
-    - example-secret
-    - other-secret
-    ```
-
-    Check the list of all available options per exporter type:
-
-    - [Deploy time](ExporterDeploytime.md#deploy-time-exporter-configuration-options)
-    - [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
-    - [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
-
-    **Secrets Examples**
-
-    1. To create a Secret named **example-secret** in **pelorus** namespace, with the option **TOKEN** with value **token_value**, using the file **secret.yaml**
-        ```yaml
-        apiVersion: v1
-        kind: Secret
-        metadata:
-          name: example-secret
-          namespace: pelorus
-        type: Opaque
-        stringData:
-          TOKEN: "token_value"
-        ```
-        run
-        ```
-        oc apply -f secret.yaml
-        ```
-        Then, add
-        ```yaml
-        [...]
-                env_from_secrets:
-                - example-secret
-        [...]
-        ```
-        to the exporter configuration.
-
-        If no `metadata.namespace` is added to Secret file, you must run the command with the namespace you want to apply it. For example, `oc apply -f secret.yaml -n pelorus`.
-
-        More [examples Secret files](https://github.com/konveyor/pelorus/tree/master/charts/secrets).
-
-    1. To create a secret named **other-secret** in **pelorus** namespace, with the
-
-        - option **SERVER** with value **server_url**
-        - option **API_USER** with value **username**
-        - option **TOKEN** with value **token_value**
-
-        run
-        ```
-        oc create secret generic other-secret -n pelorus \
-        --from-literal=SERVER=server_url \
-        --from-literal=API_USER=username \
-        --from-literal=TOKEN=token_value
-        ```
-        Then, add
-        ```yaml
-        [...]
-                env_from_secrets:
-                - other-secret
-        [...]
-        ```
-        to the exporter configuration.
+: **Recommended for sensitive data**
+
+: List of secrets, like in the following example.
+```yaml
+env_from_secrets:
+- example-secret
+- other-secret
+```
+[More examples](#secrets).
```

Comment:
I like how this was moved, but the "More examples" is very much hidden now in the config option, so it's easy to miss this part, which I think it's important enough to be better visible.

Reply:
Ok, I will try to add it were it was

Reply:
Added more highlight to the text